definitions/operations.toml

```diff
@@ -286,3 +286,18 @@ will write data into a file.
   - Service that doesn't have native support for `overwrite` SHOULD check and delete the object if exists.
 - A successful write operation SHOULD be complete, which means the object's content and metadata should be the same as specified in write request.
 """
+
+[linker]
+description = "is the interface for link"
+
+[linker.op.create_link]
+params = ["path", "target"]
+results = ["o"]
+description = """
+Will create a link object.
+
+# Behavior
+
```

Comment:
Let's add more details about the behavior for `create_link`.

- Can `path` and `target` be relative or absolute?
- What happened if `target` exists?
- What happened if `path` not exists?
- What the returning object mode will be?